appveyor.yml

```diff
@@ -0,0 +1,13 @@
+version: 3.0.{build}-pre
```

Comment:
The `-pre` isn't needed here; check out https://github.com/serilog/serilog/blob/dev/appveyor.yml#L1

When you create a `dev` branch to merge this into, the build script will pick up that it's not on master and will automatically append the `-branch-nnnnn` part.